Tasks/AzureKeyVaultV1/operations/KeyVault.ts

```diff
@@ -90,10 +90,13 @@ export class KeyVault {
         // Key vault task explicitly handles multi line masking - hence setting SYSTEM_UNSAFEALLOWMULTILINESECRET to true
         tl.setVariable("SYSTEM_UNSAFEALLOWMULTILINESECRET", "true");
 
-        if (downloadAllSecrets) {
-            return this.downloadAllSecrets(secretsToErrorsMap);
-        } else {
-            return this.downloadSelectedSecrets(this.taskParameters.secretsFilter, secretsToErrorsMap);
+        if (downloadAllSecrets)
+        {
```

Comment:
Could you please run format the document? (e.g. "alt+shift+f") 
Curly braces in TypeScript should be same level with "if" or similar conditions.

Reply:
Done